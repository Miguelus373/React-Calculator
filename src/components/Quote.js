import React from 'react';

const Quote = () => (
  <p className="quote">
    Mathematics may not teach us to add love or subtract hate,
    but it gives us hope that every problem has a solution.
    <strong> -Anonymous</strong>
  </p>
);

export { Quote as default };
