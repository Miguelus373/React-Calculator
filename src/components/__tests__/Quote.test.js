import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Quote component', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
