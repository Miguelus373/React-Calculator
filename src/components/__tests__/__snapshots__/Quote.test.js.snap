// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component Renders correctly 1`] = `
<p
  className="quote"
>
  Mathematics may not teach us to add love or subtract hate, but it gives us hope that every problem has a solution.
  <strong>
     -Anonymous
  </strong>
</p>
`;
